godotenv_test.go

```diff
@@ -98,6 +98,30 @@ func TestReadPlainEnv(t *testing.T) {
 	}
 }
 
+func TestReadPlainEnvWithLongLine(t *testing.T) {
+	envFileName := "fixtures/long.env"
+	expectedBigValue := strings.Repeat("abcd", 20000)
+	expectedValues := map[string]string{
+		"BIG_VALUE": expectedBigValue,
+		"SMALL_VALUE": "abcd",
+	}
+
+	envMap, err := Read(envFileName)
+	if err != nil {
+		t.Error("Error reading file -> " + err.Error() )
```

Comment:
The bug can be reproduced by reverting the change in `godotenv.go` and executing this new test.